### ISSUE TITLE
First time install, runtime shader compiler installation loop

### ISSUE THREAD

**Issue by robert-spurrier** [Rinnegatamante/VitaDB-Downloader#31]
Hi! Trying to install the latest release, each time I open the installed VPK I get:

`Runtime shader compiler (libshacccg.suprx) is not installed. VitaDB Downloader will proceed with its extraction`.

It looks it then proceeds to download and install the dependencies. However when I attempt to start VitaDB again, the dialogue pops up again as if nothing has happened.

I am new to Vita homebrew so apologies if I'm missing something obvious.

I used this to jailbreak: https://www.youtube.com/watch?v=zBF2ZWXRU64
And this to set up my SD2Vita: https://www.youtube.com/watch?v=8uJ_yc3zgJo

Thanks for your time and dedication to the community

**Comment by Rinnegatamante** [comment]
Do you have ur0:data/libshacccg.suprx? If not, post your ur0:tai/config.txt (or ux0:tai/config.txt if you use that instead).

**Comment by robert-spurrier** [comment]
I do not have: `ur0:data/libshacccg.suprx`

My `ux0:tai/config.txt` is actually an empty folder, not sure if that is a problem.


My `ur0:tai/config.txt` is as follows:

```
# ITLS
*KERNEL
ur0:tai/storagemgr.skprx
ur0:tai/itls.skprx

# YAMT
*NPXS10015
ur0:tai/yamt.suprx
*KERNEL
ur0:tai/yamt_helper.skprx

# This file is used as an alternative if ux0:tai/config.txt is not found.
# For users plugins, you must refresh taiHEN from HENkaku Settings for
# changes to take place.
# For kernel plugins, you must reboot for changes to take place.
*KERNEL
# henkaku.skprx is hard-coded to load and is not listed here
ur0:tai/0syscall6.skprx
ur0:tai/nonpdrm.skprx
ur0:tai/repatch.skprx
ur0:tai/PSVshell.skprx
*main
# main is a special titleid for SceShell
ur0:tai/henkaku.suprx
*NPXS10015
# this is for modifying the version string
ur0:tai/henkaku.suprx
*NPXS10016
# this is for modifying the version string in settings widget
ur0:tai/henkaku.suprx

# Optional QoL plugins
*KERNEL
# removes brightness and wifi restrictions for games
ur0:tai/NoPowerLimits.skprx
# increases max/decreases min brightness, disabled by default
# ur0:tai/vitabright.skprx
*main
# shows battery percentage near the gauge
ur0:tai/shellbat.suprx
# screenshots are png instead of jpg
ur0:tai/pngshot.suprx
```

**Comment by robert-spurrier** [comment]
I was able to accomplish installing VitaDB by installing the shader compiler manually using this method: https://cimmerian.gitbook.io/vita-troubleshooting-guide/shader-compiler/extract-libshacccg.suprx


In case its helpful, I noticed the automatic extraction process only tried to install 2 PSM runtimes (1.00, 2.01), while the manual process mentioned you MUST have 1.00, 2.00, and 2.01. 

I'm gonna close this, but if you want to use me as a guinea pig to work through any possible bugs in the automatic install, I'm happy to help out just open it back up. Thanks again.
